src/installer/pkg/projects/netcoreapp/pkg/legacy/Microsoft.NETCore.App.pkgproj

```diff
@@ -7,8 +7,31 @@
     -->
     <BuildLineupPackage>false</BuildLineupPackage>
 
+    <ShouldVerifyClosure>true</ShouldVerifyClosure>
+
     <FrameworkListTargetPath>data/</FrameworkListTargetPath>
     <SkipBuildOnRuntimePackOnlyOS>true</SkipBuildOnRuntimePackOnlyOS>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- netstandard will have cycles because OOB packages target netstandard and are used in netstandard closure,
+       We do ensure that netstandard inside the shared framework has no cycles, but we permit them in packages in order to allow for simpler netstandard-based builds.-->
+    <IgnoredReference Include="netstandard" />
```

Comment:
I don't think we should ignore this one.  We weren't before.
https://github.com/dotnet/corefx/blob/d6302a72f8eafa326d7572a02acf8f3021ebb9e8/pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj#L18-L49

We only ignore it when examining closure of standalone-packages.  When examining the closure of netcoreapp we want it to have zero cycles.

Reply:
You may encounter a failure, and that should be fixed by adding a netcoreappcurrent configuration for the lib that causes the cycle.

Reply:
I agree with Eric. Ignoring `netstandard` would cause us to miss all of the failures we've been seeing recently use to regressions from having this verification disabled.

Reply:
Thanks - I'll remove it - I think I've seen issues with netstandard cycle in my Linux build - will double-check.

Reply:
This is fixed with latest commit.